di-ipv-credential-issuer-stub/src/main/resources/data/criStubData.json

```diff
@@ -1302,6 +1302,40 @@
           }
         ]
       }
+    },
+    {
+      "criType": "National Insurance Number Verification (Stub)",
+      "label": "Alice Parker (Valid) National Insurance",
+      "payload": {
+        "name": [
+          {
+            "nameParts": [
+              {
+                "value": "Alice",
+                "type": "GivenName"
+              },
+              {
+                "value": "Jane",
+                "type": "GivenName"
+              },
+              {
+                "value": "Parker",
+                "type": "FamilyName"
+              }
+            ]
+          }
+        ],
+        "birthDate": [
+          {
+            "value": "1970-01-01"
+          }
+        ],
+        "socialSecurityRecord": [
+          {
+            "personalNumber": "AA000003D"
+          }
+        ]
+      }
     }
```

Comment:
Also can we add a success and fail evidence block in the criStubEvidencePayloads.json file - like the examples here: https://docs.google.com/document/d/19MssTJO1JxmHw_pYrRNkSvwfQdFdxuoEtx0m0uqKbsE/edit#heading=h.6tqvu5u4vw5l
Should just have checkDetails/failedCheckDetails (no scores)